### DISCUSSION TITLE
parquet file parallel read settings

### DISCUSSION THREAD

**Discussion by jiangjiguang** [trinodb/trino#13635] [Q&A]
I have a parquet file whose size is 251.86MB, the block is 128MB. so the file has two "Row group".

The problem is: no matter how much drivers I set. has two drivers to read the file, each driver reads one   Row group.

I have read the code, find the problem below：
![image](https://user-images.githubusercontent.com/12368495/184318517-55031110-ae42-4a6d-8366-3570e9e2cce1.png)

How can I set up multiple drivers read one row group?

**Comment by raunaqmorarka** [comment]
We can't create multiple splits from a single row group in the current code. In theory it might be possible to divide a row group into smaller chunks by looking at column level page offsets, but it would significantly complicate the reader code.
What's the problem you're trying to solve ?
